_posts/2019-10-04-typescript-types-vs-interfaces.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+current: post
+cover: assets/images/posts/2019-10-04-typescript-types-vs-interfaces/header.jpg
```

Comment:
Use the publish date (in this case 2019-10-09

Reply:
Changed